runtime/vm/JFRChunkWriter.cpp

```diff
@@ -109,10 +109,10 @@ VM_JFRChunkWriter::writeJFRMetadata()
 	}
 
 	/* add metadata size */
-	_bufferWriter->writeLEB128PaddedU32(dataStart, _bufferWriter->getCursor() - dataStart);
+	writeEventSize(dataStart);
 
 	if (_debug) {
-		j9tty_printf(PORTLIB, "Metadata size = %d, fromLEB128 =%u\n", (int)(_bufferWriter->getCursor() - dataStart), VM_BufferWriter::convertFromLEB128ToU32(dataStart));
+		j9tty_printf(PORTLIB, "Metadata size = %zu, fromLEB128 =%u\n", _bufferWriter->getCursor() - dataStart, VM_BufferWriter::convertFromLEB128ToU32(dataStart));
```

Comment:
This line (and several other calls to `j9tty_printf()`) are too long; please fold.

Reply:
There's a missing space after `=` in the format here.

Also the cast to `int` and the use of `%d` should be fixed on line 101.